### ISSUE TITLE
UsdFuturesApi how to liquidate assets into USDT

### ISSUE THREAD

**Issue by prozerran** [JKorf/Binance.Net#1186]
For Spot, its fine, i get convert the assets into USDT by calling PlaceOrderAsync

But for futues, calling PlaceOrderAsync trying to liquidate all my assets into USDT by placing a market sell order is not what i want. It is instead placing a SHORT order where i have a position instead,.

What i want is i already have the asset such as ETH and i just want to liquidate all my ETH into USDT.
Is there a way to do this on Futures? I can do on Spot, but futures, it just opens a short position.

Basically, it should use the Binance Convert functionality to convert my ETH back to USDT, but where's the API to do it?

**Comment by HypsyNZ** [comment]
You have to request access to the `Convert API` which is probably why it isn't part of this library.

You should read the notes of the `Convert API` if you use it in any way you might actually want to use it then Binance might at their sole discretion revoke your access

**Comment by prozerran** [comment]
ok closing
